rotkehlchen/data/all_assets.json

```diff
@@ -9977,6 +9977,15 @@
         "symbol": "QTUM",
         "type": "ethereum token"
     },
+    "QUICK": {
+        "coingecko": "quick",
+        "ethereum_address": "0x6c28AeF8977c9B773996d0e8376d2EE379446F2f",
+        "ethereum_token_decimals": 18,
+        "name": "Quickswap",
+        "started": 1602167876,
```

Comment:
Where did you take this number from? Shouldn't it be `1602175076`?

Yours seems to be off by 2 hours, which is the difference between GMT and CET in that time. Perhaps the tool you used , gave you local time?

Deployment should be `Oct-08-2020 04:37:56 PM +UTC` according to https://etherscan.io/tx/0x85d8995420d7c72fdbad750ba23869c41970f16533180cd6902f0a2fa5bb2acb

Reply:
You're right. Rookie mistake 😅